### DISCUSSION TITLE
get file declaration

### DISCUSSION THREAD

**Discussion by 567MEI13** [zeux/pugixml#618] [Q&A]
how can i get the file declaration 
![image](https://github.com/zeux/pugixml/assets/88084071/c946595f-cb0b-459e-b4f2-446b335183cd)

**Comment by zeux** [comment]
Pass `pugi::parse_default | pugi::parse_declaration` as options to `xml_document::load_file` (or any other load function) and you'll get a `node_declaration` node as the first node in the document.

**Reply by 567MEI13** [reply]
thank you！
